### ISSUE TITLE
Systematize presentation of additional info for (inflected) word form and dictionary entry

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#399]
Currently, we show the inflectional category (Verb, Noun, Particle as the general classes, and VTA-1-5 etc. as the specific subcategory) in conjunction with the inflected form, when that information comes from the dictionary source and actually concerns the dictionary entry head. For the inflectional subcategory information presented in conjunction with the dictionary head it would be useful to have an `(i)` button to show additional information.

On the other hand, the `(i)` button with the morphological/inflectional breakdown is shown in the appropriate place next to the inflected wordform, and should be kept there.

So, we should modify and extend the current presentation to something like the following (note that we are showing the word-form with morphological boundaries, which has its own separate issue):

![image](https://user-images.githubusercontent.com/15003407/80675678-51b5df80-8a72-11ea-9ff6-a0f8ceecde39.png)

In the above, 1) _ni·kî-nitawi-wâpam·âw_ is the inflected word-form, for which the 2) linguistic inflectional breakdown applies (as is currently the case). In contrast, 3) _wâpamêw_ is the dictionary entry head, for which the 4a) general (`Verb`) and 4b) specific inflectional subcategory (`like: wîcihêw`, using the plain English label) information applies. In the underlying database, 3a is specified by labels, in this case `VTA-1`. However, `crk.altlabel.tsv` contains alternative relabelings, of which by default we would want to use the plain English one (here: `like: wîcihêw`). However, we could allow for providing additional information, say the code (`VTA-1`) as well as the long linguistic label (`transitive animate verb – class 1: regular`) by using a `(i)` button, e.g. showing these together as `transitive animate verb – class 1: regular (VTA-1)` or `VTA-1: transitive animate verb – class 1: regular`

As a glimpse of further development later on, through settings or otherwise, one might want to be able to choose also the full linguistic (`transitive animate verb – class 1: regular`) or plain nêhiyawêwin (`tâpiskôc: wîcihêw`) labels as the primary ones shown.

**Comment by eddieantonio** [comment]
I like this a lot!

**Comment by aarppe** [comment]
Older issues #232 and #296 are earlier takes on this issue, and might be partially outdated.

**Comment by eddieantonio** [comment]
@kobexamoh this is for you!

**Comment by eddieantonio** [comment]
@kobexamoh, for the time being, I've got this, but I'll let you know if things change

**Comment by kobexamoh** [comment]
Thanks Eddie!

**Comment by eddieantonio** [comment]
@aarppe, what should this look like for the lemma?

**Comment by aarppe** [comment]
Good question! I had focused so much on the inflected forms, that I had ignored what this would look like for the lemma. The lemma is still an inflected form, so we do want to provide its linguistic analysis/breakdown. The lemma also is associated with the general word class and specific inflectional subcategory, which we would want to show, too, for consistency.

I can think of two options, depending on the space that is available:

![image](https://user-images.githubusercontent.com/15003407/83898146-c33d1980-a713-11ea-9060-94714b008c31.png)

_OR_

![image](https://user-images.githubusercontent.com/15003407/83898188-d819ad00-a713-11ea-8386-92702fa1b55f.png)

Especially in the option on presenting all on the same line, this could be seen to further motivate having for the inflectional subcategory information a button different from ![image](https://user-images.githubusercontent.com/15003407/83899154-2da28980-a715-11ea-8f6e-e17081be7f4c.png) that gives the inflectional linguistic analysis/breakdown  - that's why I'm using the red question-mark ![image](https://user-images.githubusercontent.com/15003407/83899056-0b107080-a715-11ea-8427-45856345b283.png) button above. But this is just a very quick stab at what the solution could be.

Furthermore, the emojis would allow the representation of some information in a more denser space, but they would currently not exactly map one-to-one to what I've thought above, as they would represent the _specific word-class_, which we would not be showing currently - and which would be difficult to express tersely in words (and we don't want to get a sequence of similar, repetitive, long labels, e.g. `like: wîcihêw -- like: itêw`. I suppose we could choose that without emojis, the general word class is shown; with emojis, the specific word class.
